data/sql/updates/pending_db_world/rev_1560493947655900000.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1560493947655900000');
+
+UPDATE `spell_script_names` SET `spell_id`=-6229 WHERE `ScriptName`='spell_warl_shadow_ward';
```

Comment:
```suggestion
UPDATE `spell_script_names` SET `spell_id`=6229 WHERE `ScriptName`='spell_warl_shadow_ward';
```

Reply:
@BarbzYHOOL We need that minus to check for all ranks of the spell. Without it the spell script will only work for rank 1

Reply:
@BarbzYHOOL no, it must be with - to check ranks. Just like @lineagedr explained

Reply:
ok thank you, i suspected it was not a mistake but i wasn't sure :P